### PR TITLE
Display similar posts by using tag 

### PR DESCRIPTION
**Pull request by Karthikeyantestpress**

### CODE REVIEW

File: mysite/blog/views.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
 from django.shortcuts import render, get_object_or_404
 from .models import Post
-from django.views.generic import ListView ,FormView
-from .forms import EmailPostForm ,CommentForm
+from django.views.generic import ListView, FormView
+from .forms import EmailPostForm, CommentForm
```
Comment (waranlogesh) [inline mysite/blog/views.py]: Not related to this PR.

File: mysite/blog/views.py [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 from django.core.mail import send_mail
 from django.urls import reverse_lazy
 from django.contrib.messages.views import SuccessMessageMixin
 from taggit.models import Tag
 
 
-
```
Comment (waranlogesh) [inline mysite/blog/views.py:13]: .

File: mysite/blog/views.py [hunk 3]
```diff
@@ -62,9 +61,11 @@ def post_detail(request, year, month, day, post):
             "comments": comments,
             "new_comment": new_comment,
             "comment_form": comment_form,
+            "similar_posts": post.get_top_four_similar_posts(),
         },
     )
 
+
```
Comment (waranlogesh) [inline mysite/blog/views.py:67]: .

File: mysite/blog/views.py [hunk 4]
```diff
@@ -82,6 +83,7 @@ def get_context_data(self, **kwargs):
         data["tag"] = self.tag
         return data
 
+
```
Comment (waranlogesh) [inline mysite/blog/views.py:85]: Formatting changes should be a separate PR or you can push them to master directly

**Review by waranlogesh — CHANGES_REQUESTED** [review]

File: mysite/blog/views.py (hunk 1 above)
Comment (Karthikeyantestpress) [reply mysite/blog/views.py]: this change has been done in the appropriate Pr and rebased
